test/docker-volume-test.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+. $(dirname "${BASH_SOURCE[0]}")/common.sh
+
+# prepare RUNU_AUX_DIR
+create_runu_aux_dir
+
+DOCKER_RUN_ARGS="run --rm -i --runtime=runu-dev --net=none $DOCKER_RUN_ARGS_ARCH"
+
+if [ $TRAVIS_OS_NAME = "linux" ] ; then
+
+fold_start test.docker.1 "docker-volume: naive -v option for directory"
+
+MNT_SRC=$PWD/host_dir
+MNT_DST=/mnt
+mkdir -p $MNT_SRC
+touch    $MNT_SRC/foo
+touch    $MNT_SRC/bar
+docker $DOCKER_RUN_ARGS -e RUMP_VERBOSE=1 \
+       -e LKL_CONFIG=/lkl.json \
+       -e LKL_USE_9PFS=1  -v $MNT_SRC:$MNT_DST \
+       thehajime/runu-python:$DOCKER_IMG_VERSION \
+       python -c "import os; print(os.listdir('/mnt'))" | egrep "foo.*bar|bar.*foo"
+rm -rf $MNT_SRC
+
+fold_end test.docker.1
+
+fold_start test.docker.2 "docker-volume: naive -v option for file"
+
+MNT_SRC=/tmp/hello.txt
+MNT_DST=/mnt/hello.txt
+echo "hello_world" > $MNT_SRC
+docker $DOCKER_RUN_ARGS -e RUMP_VERBOSE=1 \
+       -e LKL_USE_9PFS=1  -v $MNT_SRC:$MNT_DST \
+       -e RUNU_AUX_DIR=$RUNU_AUX_DIR alpine /bin/busybox cat /mnt/hello.txt | grep 'hello_world'
+
+rm -f $MNT_SRC
+fold_end test.docker.2
+
+fold_start test.docker.3 "docker-volume: naive -v option for named volume"
+
+docker run --rm -v named_vol:/mnt alpine touch /mnt/foo /mnt/bar
+docker $DOCKER_RUN_ARGS -e RUMP_VERBOSE=1 \
+       -e LKL_CONFIG=/lkl.json \
```

Comment:
maybe, we don't need to use LKL_CONFIG ?
(because CI reports `no netdev available (null)` , https://travis-ci.org/github/ukontainer/runu/jobs/730144670#L1877)